### ISSUE TITLE
Implement GoogLeNet architecture

### ISSUE THREAD

**Issue by dagleaves** [dagleaves/VisionNETs#5]
https://arxiv.org/abs/1409.4842

**Comment by dagleaves** [comment]
The model output format will have to change to account for the auxiliary logits produced by GoogLeNet. Possible solutions include `namedtuple` for all models. If the aux logits are `None` then they can be ignored. Otherwise, compute the `0.3 * loss` weighting for the auxiliary losses according to the GoogLeNet paper.

The only consideration is that the default values must be `None` for the aux logits. That will determine the data structure to use.
